package.json

```diff
@@ -111,6 +111,9 @@
     },
     "topicSeparator": " ",
     "topics": {
+      "attest": {
+        "description": "[Attest]       Attest to 'Not Reviewed' controls in order to account for all of your controls"
```

Comment:
My attempt at making this a little more clear. Anyone else can give it a try as well.
`Attest to 'Not Reviewed' controls by creating attestation data and applying attestation data to your results`

Reply:
Your suggestion feels like a circular definition but I do agree that what is there could be significantly improved.  Maybe @ejaronne has a suggestion.